src/util.js

```diff
@@ -105,9 +105,19 @@ function serializeArgs<T>(args : $ReadOnlyArray<T>) : string {
     try {
         // $FlowFixMe[method-unbinding]
         return JSON.stringify(Array.prototype.slice.call(args), (subkey, val) => {
+
             if (typeof val === 'function') {
                 return `memoize[${ getObjectID(val) }]`;
             }
+
+            if (val instanceof window.Element) {
+                return {};
+            }
+        
+            if (val !== null && typeof val === 'object' && val.nodeType === 1 && typeof val.style === 'object' && typeof val.ownerDocument === 'object') {
```

Comment:
@bluepnume Wow!  Can we explain all of this in a method that describes it?  Non-blocking, but would be easier to understand what this condition is really doing.

Reply:
Yeah, this is just copy-pasted from `dom.js` `isElement` to avoid a circular dependency. Will definitely add a comment to explain.

Reply:
I agree it would be nice to explain this check in a comment. From my understanding this is verifying if the object is an HTMLElement which would have `ownerDocument`, `style`, and `nodeType` attributes. And if it is we safely return an object rather than trying to call `JSON.stringify()` on it.

Reply:
Updated 👍